src/app.ts

```diff
@@ -137,6 +137,11 @@ function createWindow(): void {
   if (is.macos) {
     app.dock.setMenu(buildDockMenu(mainWindow))
   }
+
+  // TODO: create ipcMain.on(..) events for opening inbox,compose, and sent views
+  ipcMain.on('open-inbox-view', () => {
```

Comment:
@timche 

I've started on the `ipc` method of doing this as you suggested, but I'm running into an issue where `ipc` here is always undefined. It looks like the import isn't working properly. I've ran a few searches but they all seemed to point me toward
 
* removing `node_modules` and reinstalling (tried that)
* not correctly importing it (followed the documentation)

I've also noticed that you use it in `preload.ts`, but I haven't been able to confirm whether it works there or not, since I don't think it's actually being run/used AFAIK.

Any thoughts?